pkg/ddevapp/ddevapp.go

```diff
@@ -32,7 +33,7 @@ import (
 	"github.com/drud/ddev/pkg/output"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/drud/ddev/pkg/version"
-	"github.com/fsouza/go-dockerclient"
+	docker "github.com/fsouza/go-dockerclient"
```

Comment:
Seems to still compile, but one of the tools vscode uses under the hood made this change. Seems to still compile, so maybe this is a desirable change?